### PR TITLE
Added instructions for installing Node

### LINKED ISSUE #2038: Add instructions to use `volta` to install node in the READMe
## Problem

We've gotten at least two newer contributors confused about which Node version to use. We pin the node version using Volta, but do not specifically direct the reader of the README to initialise volta in the repository. This seems to be a gap in the assumptions of what we think people will know on their own.

## Description

Add a new bit to the README before the `pnpm` installation instructions to run `volta install node@16.15.0`. I believe there's a way to get volta to automatically install Node at the pinned version (I thought that was the entire point) but I haven't found it in the docs yet and I don't have time right now to research it further. If whoever is working on this can figure that out, make sure the instructions specify to follow that method. Maybe it has something to do with using `volta run pnpm dev` instead of `volta run pnpm`? Additionally, we should add to check the package.json volta pinned version to see which version of Node the project currently expects (if that is indeed our canonical node version for the project).

## Implementation

- [ ] 🙋 I would be interested in implementing this feature.

### PR DESCRIPTION
**Pull request by rajdeepdas2000**
## Fixes

Fixes #2038 by @sarayourfriend

## Description
Added instructions for installing volta, node and pnpm

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [x] My pull request targets the *default* branch of the repository (`main`) or a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Review by sarayourfriend — DISMISSED** [review]

**Review by obulat — CHANGES_REQUESTED** [review]
Looks great, except for one change I suggested inline.

File: README.md [hunk 1]
```diff
@@ -33,7 +39,7 @@ Run the following commands in order to have the code up and running on your mach
 ```bash
 # Builds and serves assets with hot-reload
 # Automatically invokes pnpm install and pnpm i18n
-pnpm dev
+Run volta run pnpm dev
```
[the remark this answers is not in the document]
Comment (rajdeepdas2000) [reply README.md]: Done.

**Review by obulat — APPROVED** [review]
Thank you for your contribution, @rajdeepdas2000 !

**Review by zackkrida — APPROVED** [review]
These instructions look good. I can't wait until pnpm is officially supported to make this simpler. Volta automaticaly symlinks the correct node version, but it only does its check to create the correct symlink the first time `node` or `npm` is evoked in a directory. So if someone has volta installed, clones the frontend, and runs a command like`node -v` in the repo root, at that point volta will automatically install and symlink node `v16.15.0` before evoking the command.

Once pnpm support is official, running `pnpm dev` will be sufficient to install and link the correct node version. At that time we can remove these instructions.

**Pull request merged**